docs/getting_started.rst

```diff
@@ -7,9 +7,16 @@ Getting Started
 Features
 ========
 
-**niveristand** allows you to:
+**niveristand** has two major capabilities: system definition and real-time sequence scripting. NI recommends you use an editor with code completion, such as `Visual Studio Code <https://code.visualstudio.com/docs/languages/python/>`_, to make it easier to browse and use this code.
 
-- Convert Python functions into NI VeriStand real-time (RT) sequences that are compatible with the NI VeriStand engine and Stimulus Profile Editor and save the converted Python functions to a file.
+Scripting system definition files
+---------------------------------
+You can script system definitions (.nivssdf) files for use in the NI VeriStand editor and deployable to the NI VeriStand engine.
```

Comment:
"You can script system definition (.nivssdf) files for use in the NI VeriStand editor and deploy them to the NI VeriStand engine."